Mercury-Store/Mercury-Store/Home/ViewModels/HomeViewModel.swift

```diff
@@ -6,8 +6,15 @@
 //
 
 import RxSwift
+protocol HomeFlow: AnyObject {
+    func goToCategoriesTab(with itemName: String)
+    func goToBrandDetails(with brandItem: SmartCollection)
+}
 
 struct HomeViewModel {
```

Comment:
Please take care of ordering
// MARK: Properties
// MARK: Lifecycle - Init
// MARK: Methods